class.jetpack.php

```diff
@@ -446,6 +446,10 @@ private function __construct() {
 		add_action( 'deleted_user', array( $this, 'unlink_user' ), 10, 1 );
 		add_action( 'remove_user_from_blog', array( $this, 'unlink_user' ), 10, 1 );
 
+		// define a few REST endpoints for getting through the connection process
+		// without XMLRPC
+		add_action( 'rest_api_init', array( $this, 'rest_api_init' ) );
```

Comment:
Shouldn't this simply use the `Jetpack_Core_Json_Api_Endpoints::register_endpoints()` class we already have available in https://github.com/Automattic/jetpack/blob/ea8c5530536b05f80cfac4f4c42380c4a8554504/_inc/lib/class.core-rest-api-endpoints.php#L19 ?